.github/workflows/build.yml

```
@@ -9,3 +9,13 @@ jobs:
       - uses: psf/black@stable
         with:
           options: "--check"
+  jsonlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)